eo-maven-plugin/src/main/java/org/eolang/maven/EmptyDirectoriesIn.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+package org.eolang.maven;
+
+import com.jcabi.log.Logger;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Delete empty directories in provided root.
+ *
+ * @since 0.55
+ */
+final class EmptyDirectoriesIn {
+    /**
+     * Root path.
+     */
+    private final Path root;
+
+    /**
+     * Ctor.
+     * @param root Root directory.
+     */
+    EmptyDirectoriesIn(final Path root) {
+        this.root = root;
+    }
+
+    /**
+     * Clear empty directories in {@code this.root}.
+     */
+    void clear() {
+        if (Files.isDirectory(this.root)) {
```

Comment:
@maxonfjvipon in general, it's better to avoid this: https://www.yegor256.com/2015/01/21/if-then-throw-else.html Better throw first and then continue as normal.